packages/core/src/com/communication.ts

```diff
@@ -82,41 +88,50 @@ export class Communication {
     private disposeListeners = new Set<(envId: string) => void>();
     private callbackToEnvMapping = new Map<string, string>();
     private messageIdPrefix: string;
+    // manual DEBUG_MODE
+    private DEBUG = false;
     constructor(
-        host: Target,
+        private host: Target,
```

Comment:
Its strange to have public/private and non-annotations together? are we sure no one publicly uses the host?

Reply:
this turns it into a member. before it wasn't even a member